Makefile

```diff
@@ -55,6 +55,8 @@ default: test
 BUILD_DEPS += Makefile
 BUILD_DEPS += CMakeLists.txt
 
+SKIP_DOCS := false
```

Comment:
Does this need to be `?=`?

Reply:
It *should* be for sure, but it worked in testing…